test/models/asyncapi_test.js

```diff
@@ -978,6 +984,88 @@ describe('AsyncAPIDocument', function() {
   });
   /* eslint-enable sonarjs/cognitive-complexity */
 
+  describe('#stringify()', function() {
+    it('should stringify simple document', async function() {
+      const doc = await parser.parse(simpleInputJSON, { path: path.join(__filename, '../../') });
+      const stringified = AsyncAPIDocument.stringify(doc);
+      expect(stringified).to.be.equal(simpleOutputJSON);
```

Comment:
I wished we used snapshots here, trying to look at the output JSON in a oneliner is 😅  😆

Reply:
We do not use any snapshots in the unit tests of functions. I know this can be tiresome, but I am consistent with the other tests in this repository.

Reply:
I know, it was only a comment not a suggestion to change 😄